session_4/blog/app/models/ability.rb

```diff
@@ -35,12 +35,18 @@ def initialize(user)
 
       # Can destroy their own profile
       can :destroy, User, id: user.id
-            
+      
+      #cann like article
+      can :like, Article
+      can :unlike, Article
+      
       #admin
       if user.admin
         can :edit, Article
         can :update, Article
         can :destroy, Article
+        can :like, Article
```

Comment:
You don't have to repeat these permissions again - they will be applied to both admin and non-admin users